### PR TITLE
Provide example of complex where condition

### PR DESCRIPTION
**Pull request by oaksofmamre**
Also provide link to "Query Options" on the same page. Removed inaccurate quotes for (nin, in, contains, startsWith, endsWith). Made code format consistent in the "Criteria Modifiers" section. Added short note that query options are used in conjunction with 'where'

### CODE REVIEW

**Comment by mikermcneil** [comment]
@oaksofmamre Thank you!  This is a good improvement.  Reading this made me realize we should change "Query options" to be "Criteria clauses" -- otherwise it's confusing with the other concept of query options in the world of blueprints.  Let's tackle that on this branch when we get a chance

**Comment by oaksofmamre** [comment]
hey @mikermcneil terms updated from "Query options" to "Criteria clauses"

File: concepts/ORM/Querylanguage.md [hunk 1]
```diff
@@ -8,16 +8,11 @@ The syntax supported by Sails' model methods is called Waterline Query Language.
...
+Queries can be built using either a `where` key to specify attributes, or excluding it.
 
-```javascript
-
-var peopleNamedMary = await Model.find({
-  name: 'mary'
-});
-
-// OR
+Using the `where` key allows you to also use [criteria clauses](#criteria_clauses), such as `limit`, `skip`, and `sort`.
```
Comment (rachaelshaw) [inline concepts/ORM/Querylanguage.md]: When the docs are recompiled for the Sails website, all the headings are automatically turned into links, so there's no need to manually set one up here -- before I merge this in, would you change this link to point at https://sailsjs.com/documentation/concepts/models-and-orm/query-language#?criteria-clauses, then remove that link below and just leave it as a regular header?

Comment (oaksofmamre) [reply concepts/ORM/Querylanguage.md]: @rachaelshaw this should be resolved now thx

**Comment by rachaelshaw** [comment]
Thanks @oaksofmamre! Sorry it took forever to actually merge this.

**Pull request merged**